docs/admin/resourcequota/index.md

```diff
@@ -84,45 +85,77 @@ You might want to set a pods quota on a namespace
 to avoid the case where a user creates many small pods and exhausts the cluster's
 supply of Pod IPs.
 
+## Quota Scopes
+
+Each quota can have an associated set of scopes.  A quota will only measure usage for a resource if it matches
+the intersection of enumerated scopes.
+
+When a scope is added to the quota, it limits the number of resources it supports to those that pertain to the scope.
+Resources specified on the quota outside of the allowed set results in a validation error.
+
+| Scope | Description |
+| ----- | ----------- |
+| Terminating | Match pods where `spec.activeDeadlineSeconds >= 0` |
+| NotTerminating | Match pods where `spec.activeDeadlineSeconds is nil` |
+| BestEffort | Match pods that have best effort quality of service. |
+| NotBestEffort | Match pods that do not have best effort quality of service. |
+
+A `BestEffort` scope restricts a quota to limit the following resources:
+* pods
+
+A `Terminating`, `NotTerminating`, and `NotBestEffort` scope restricts a quota to tracking the following resources:
+* cpu
+* limits.cpu
+* limits.memory
+* memory
+* pods
+* requests.cpu
+* requests.memory
+
+## Requests vs Limits
+
+When allocating compute resources, each container may specify a request and a limit value for either CPU or memory.
+The quota can be configured to quota either value.
+
+If the quota has a value specified for `requests.cpu` or `requests.memory`, then it requires that every incoming
+container makes an explicit request for those resources.  If the quota has a value specified for `limits.cpu` or `limits.memory`,
+then it requires that every incoming container specifies an explict limit for those resources.
+
 ## Viewing and Setting Quotas
 
 Kubectl supports creating, updating, and viewing quotas:
 
 ```shell
-$ kubectl namespace myspace
-$ cat <<EOF > quota.json
-{
-  "apiVersion": "v1",
-  "kind": "ResourceQuota",
-  "metadata": {
-    "name": "quota"
-  },
-  "spec": {
-    "hard": {
-      "memory": "1Gi",
```

Comment:
I am trying to encourage using multiple quota documents to split compute-resource quotas from object count quotas.  I have a separate PR that adds an example of that, but I think for this scenario, I will add the compute resources section.